cli/tests/testdata/compile/workers/basic.js

```diff
@@ -0,0 +1,11 @@
+import "./worker.js";
```

Comment:
I ran into typechecking errors when I tried to make this a TS file, since it seems like the typechecker is detecting that `worker.ts` is both imported in a main thread context and in a worker context, and tries to apply both declaration files, which have duplicate definitions. This seems to happen even with `deno compile --no-check`, so it might be a bug.

Reply:
I checked that again, and the errors are different from what I remember. In particular, the only error is about typechecking `worker.ts` as being in the main thread, and `--no-check` doesn't error.

Typechecking `worker.ts` wrong is still a bug introduced by this PR, but I think one that can be fixed after it's merged.